src/qemu/qemu.conf

```diff
@@ -581,7 +581,7 @@
 # Warning: Disabling this option means that a compromised guest can
 # exploit the privileges and possibly do damage to the host.
 #
-#clear_emulator_capabilities = 1
+clear_emulator_capabilities = 0
```

Comment:
Are you concerned about the warning here? Is this situation handled some other way?